src/lib/userStorage/UserStorageClass.js

```diff
@@ -820,6 +820,8 @@ export class UserStorage {
     } = data
     const { address, asset, initiator, initiatorType, value, displayName, message, avatar } = this._extractData(event)
 
+    const isBridge = this.wallet.getBridgeAddresses().includes(address.toLowerCase())
```

Comment:
why not add this field to the TX record itself instead of checking for it everytime in the feed?

Reply:
could be, but we still need it here if we want older bridge transactions to reflect the change as well

or you have a different proposal how to handle those?

Reply:
we can keep this if item.isBridge is undefined. for new txs item.isBridge should be false/true

Reply:
ok done
moved the logic for formatEvents to extractData as well